src/main/java/net/minestom/server/instance/palette/Section.java

```diff
@@ -91,6 +96,13 @@ public void setBlockAt(int x, int y, int z, short blockId) {
             block ^= clear << bitIndex;
             block |= (long) blockId << bitIndex;
 
+            final boolean isCurrentAir = Block.fromStateId(getBlockAt(x, y, z)).isAir();
```

Comment:
Hmm I am not really confident performance-wise. Would probably be better to retrieve the state id from the `block` value above